up42/order.py

```diff
@@ -63,9 +70,9 @@ def status(self) -> str:
     @property
     def order_details(self) -> dict:
         """
-        Gets the Order Details.
+        Gets the Order Details. Only for tasking type orders, archive types return empty.
         """
-        return self.info["orderDetails"]
+        return self.info.get("orderDetails", {})
```

Comment:
This one is a fix to some issue I got also from our migration. order_details are only valid for tasking orders then it should raise a key error if they use this method with the current implementation over Archive orders.